07_Bootstrap/index.html

```diff
@@ -0,0 +1,166 @@
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Screenshot</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Team</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" href="#">Pricing</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link active dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                  Dropdown
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Action</a></li>
+                  <li><a class="dropdown-item" href="#">Another action</a></li>
+                  <li><hr class="dropdown-divider"></li>
+                  <li><a class="dropdown-item" href="#">Something else here</a></li>
+                </ul>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" href="#">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+    </nav>
+
+    <!-- Body -->
+    <div class="container mt-5">
+        <section class="text-center mb--md-5 pb-md-5">
+            <h1 class="fw-lighter">Simple System For Discount</h1>
+            <p class="fw-lighter">Integer Cursus Bibendum Augue Ac Cursus</p>
+            <div class="d-md-flex justify-content-center mt-5">
+                <div class="mb-5 mb-md-0">
+                    <img style="width: 50px;" src="./assets/img/svg/cloud.svg" alt="image">
+                    <div class="text-center mt-5 px-5">
+                        <p class="fw-bold">INTRODUCING WHATSAPP</p>
+                        <p class="mx-5">Lorem Ipsum is simply text of the printing and typesetting industry</p>
+                        <p class="text-secondary">Read More</p>
+                    </div>
+                </div>
+                <div class="my-5 my-md-0">
+                    <img style="width: 50px;" src="./assets/img/svg/planet.svg" alt="image">
+                    <div class="text-center mt-5 px-5">
+                        <p class="fw-bold">USER FRIENDLY INTERFACE</p>
+                        <p class="mx-5">Lorem Ipsum is simply text of the printing and typesetting industry</p>
+                        <p class="text-secondary">Read More</p>
+                    </div>
+                </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)